tests/torchtune/modules/loss/test_liger_ce_loss.py

```diff
@@ -0,0 +1,108 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+import pytest
+import torch
+import torch.nn.functional as F
+from tests.test_utils import assert_expected
+from torch import nn
+from torchtune.modules.loss import LigerFusedCrossEntropyLoss
+from torchtune.training.seed import set_seed
+
+
+class Model(nn.Module):
+    def __init__(self, vocab_size, embed_dim):
+        super().__init__()
+        self.output = nn.Linear(embed_dim, vocab_size)
+
+    def forward(self, x):
+        return self.output(x)
+
+
+@pytest.fixture(autouse=True)
+def random():
+    set_seed(42)
+
+
+class TestLigerFusedCrossEntropyLoss:
+    def test_liger_fused_cross_entropy_loss(self):
```

Comment:
Since this test requires cuda you should add the "@gpu_test(gpu_count=1)" decorator from "from tests.test_utils import gpu_test". Along with testing the loss value, I think it would be good to test a single forward and backward pass with opt step to ensure all the gradients are propagating back correctly too. You can use "fixed_init_model" (also from test_utils) as well to make it easier to initialize the model the same way each time

Reply:
I think it would be good to add `@pytest.mark.parametrize("compile", [False, True])` to the test and pass in compile as an argument on whether to call `apply_compile_strategy` on the loss